### PR TITLE
[Manta] Add utility pallet to manta runtime

### PR DESCRIPTION
**Pull request by Dengjianping**
We can send batch transactions by this pallet.
I tested it, and worked.

### CODE REVIEW

File: runtimes/manta/runtime/src/lib.rs [hunk 1]
```diff
@@ -461,6 +467,7 @@ construct_runtime!(
         Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>},
         TransactionPayment: pallet_transaction_payment::{Module, Storage},
         Scheduler: pallet_scheduler::{Module, Call, Storage, Event<T>},
+		Utility: pallet_utility::{Module, Call, Event},
```
Comment (zhenfeizhang) [inline runtimes/manta/runtime/src/lib.rs:470]: nits: there seems to be a formatting issue here. can we align the code?

**Review by zhenfeizhang — APPROVED** [review]
LGTM

File: runtimes/manta/runtime/src/lib.rs (hunk 1 above)
Comment (Dengjianping) [reply runtimes/manta/runtime/src/lib.rs:470]: Sorry, the indentation of my editor is tab, I'll replace it with space.

**Review by stechu — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**